dns/Makefile

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.EXPORT_ALL_VARIABLES:
+SHELL := /usr/bin/env bash
+GOOGLE_CLOUD_PROJECT := "kubernetes-public"
+
+default: check
+
+.PHONY: check
+check: check-canary check-prod
+
+.PHONY: check-canary
+check-canary:
+	$(CURDIR)/check-zone.sh canary.k8s.io.
+
+.PHONY: check-prod
+check-prod:
+	$(CURDIR)/check-zone.sh k8s.io.
+
+.PHONY: push-canary
+push-canary:
+	$(CURDIR)/push.sh canary.k8s.io.
```

Comment:
As far as I can tell, push.sh doesn't take any args ?

Reply:
you're right. fixing.